stl/src/atomic_wait.cpp

```diff
@@ -238,13 +245,13 @@ bool __stdcall __std_atomic_wait_indirect(const void* const _Storage, _Atomic_wa
     auto& _Entry = _Atomic_wait_table_entry(_Storage);
     switch (_Wait_context._Wait_phase_and_spin_count) {
     case _Atomic_wait_phase_wait_none:
-        _Wait_context._Counter = _Entry._Counter.load(std::memory_order_relaxed);
+        _Wait_context._Counter = _Entry._Counter.load(_STD memory_order_relaxed);

```

Comment:
I thought that `_STD` instead of `std::` is only needed in headers. Is it needed for implementation `.cpp` too?

Reply:
_Ugly isn't necessary in `.cpp` either but I think we try to follow consistent conventions. I don't think we are consistent :/

Reply:
The consistency of `stl/src` is low, since we spend 95% of our time in the headers and they contain highly templated code (where strong consistency is especially valuable). We're slowly improving the separately compiled sources. Right now I wouldn't worry too much about global consistency, but consistency within a single file is a reasonable goal.